charts/values/calico-bpf.yaml

```diff
@@ -1,8 +1,3 @@
----
```

Comment:
For all of these, I just removed the bits that told jekyll to treat this as a helm values.yaml, and left the actual values.yaml contents so we can simply pass them to helm directly.